cmd/sealer/cmd/build.go

```diff
@@ -95,8 +97,9 @@ func init() {
 	buildCmd.Flags().StringVarP(&buildConfig.BuildType, "mode", "m", "lite", "ClusterImage build type, default is lite")
 	buildCmd.Flags().StringVarP(&buildConfig.KubefileName, "kubefile", "f", "Kubefile", "Kubefile filepath")
 	buildCmd.Flags().StringVarP(&buildConfig.ImageName, "imageName", "t", "", "the name of ClusterImage")
+	buildCmd.Flags().BoolVar(&buildConfig.DownloadImage, "downloadImage", true, "download manifest image")
```

Comment:
we need more clear parameter description here：build with application container image, default value is true.